docs/source/configure.rst

```diff
@@ -51,10 +51,11 @@ isn't mentioned in the list of :ref:`settings <settings>`.
 Configuration File
 ==================
 
-The configuration file should be a valid Python source file. It only needs to
-be readable from the file system. More specifically, it does not need to be
-importable. Any Python is valid. Just consider that this will be run every time
-you start Gunicorn (including when you signal Gunicorn to reload).
+The configuration file should be a valid Python source file with a **python
+extension** (e.g. `gunicorn.conf.py`). It only needs to be readable from the
+file system. More specifically, it does not need to be importable. Any Python
```

Comment:
By "does not need to be importable" do you mean it does not have to be on the module path (`sys.path`, `PYTHONPATH`)? If so, saying that more explicitly might be clearer.